src/Components/ListOfArticles.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Grid, Container } from 'semantic-ui-react'
+
+class ListOfArticles extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      articles: []
+    }
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:3000/api/v1/articles').then(response => {
+      this.setState({ articles: response.data.entries });
+    });
+  }
+
+  render() {
+    let articleList
+    let componentArticlesState = this.state.articles.length
```

Comment:
instead of that you can simply write `if(this.state.articles.length)` this will evaluate to false if the array is empty

Reply:
done